lib/puppet/provider/certs_bootstrap_rpm/katello_ssl_tool.rb

```diff
@@ -20,9 +20,9 @@ def run
                             '--requires', 'subscription-manager',
                             '--post', post_script_file,
                             *resource[:files])
-      if resource[:alias]
+      if rpm = last_rpm && resource[:alias]
```

Comment:
Nitpick, but usually assignment in a conditional should be in parens.  Do we enforce that?